### ISSUE TITLE
Training on a smaller dataset doubt.

### ISSUE THREAD

**Issue by abhishek-96** [thunlp/TensorFlow-Summarization#10]
I don't have a Titan X and before doing the training on the whole dataset on AWS I would like to train it locally and see how the model converges. So for that apart from reducing the dataset size to 500 what all changes in the parameters are needed to be done so that I can achieve the training of the model in couple of days ?
Also could you give a link to sum_dict and doc_dict .txt files so that I can the test model according the trained weights you have already provided.
Thank You.

**Comment by leopard1** [comment]
For dict files, pls check [](http://sfo.citr.me/static/)

You can use the same 500 sentence pairs as train and test set. Try overfit the model on your laptop.

**Comment by abhishek-96** [comment]
Thanks
